migration/idempotent/001-base.sql

```diff
@@ -2068,6 +2113,7 @@ BEGIN
 
     IF NOT view_exists THEN
         EXECUTE FORMAT('GRANT SELECT ON prom_series.%1$I TO prom_reader', view_name);
+        EXECUTE FORMAT('ALTER VIEW prom_series.%1$I OWNER TO prom_modifier', view_name);
```

Comment:
(accidentally deleted James' comment in which he said) we should remove this line